### PR TITLE
Update pVACview anchor figure

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: docs/pvacview/pvacseq_module/pvacseq_vignette.rst [hunk 1]
```diff
@@ -219,7 +219,11 @@ The candidate being investigated has a good binding affinity (median IC50 score
     :alt: pVACview Vignette
     :figclass: align-left
 
-The mutation is not in an anchor position (see ``Anchor heatmap`` tab). Anchor prediction scores for each amino acid position are provided in the ``Anchor Weights`` tab at the bottom. Both mutant and wildtype peptides are good binders, yet the mutant peptide is a stronger binder. This is scenario number 2 (WT strong binder, MT strong binder, MT not in an anchor position) according to the Scenario Guide, where the neoantigen candidate is favorable and can be accepted.
+The mutation is not in an anchor position (see ``Anchor heatmap`` tab). A list of anchor positions for each allele-length combination
+is previded in the ``Anchor Positions`` panel below. Additional, the underlying anchor prediction scores for each amino acid position are
```
Comment (malachig) [inline docs/pvacview/pvacseq_module/pvacseq_vignette.rst:223]: Propose changing `Additional,` to `Additionally,`

**Review by malachig — COMMENTED** [review]
Just had one tiny wording suggestion.

File: docs/pvacview/pvacseq_module/pvacseq_vignette.rst (hunk 1 above)
Comment (susannasiebert) [inline docs/pvacview/pvacseq_module/pvacseq_vignette.rst:223]:
```suggestion
is provided in the ``Anchor Positions`` panel below. Additionally, the underlying anchor prediction scores for each amino acid position are
```

**Review by malachig — APPROVED** [review]
Looks good to me. I checked all the images before/after to confirm they are all being updated in the expected way.

**Pull request merged**